review/communication-react.api.md

```diff
@@ -0,0 +1,1828 @@
+    // (undocumented)
+    sendTypingIndicator(): Promise<void>;
+    // (undocumented)
+    setTopic(topicName: string): Promise<void>;
+}
+
+// @public
+export type ChatAdapterErrors = {
+    [operation: string]: Error;
+};
+
+// @public (undocumented)
+export type ChatBaseSelectorProps = {
+    threadId: string;
+};
+
+// @public
+export const ChatClientProvider: (props: ChatClientProviderProps) => JSX.Element;
+
+// @public (undocumented)
+export type ChatClientProviderProps = {
+    children: React_2.ReactNode;
+    chatClient: StatefulChatClient;
+};
+
+// @public (undocumented)
+export type ChatClientState = {
+    userId: CommunicationIdentifierKind;
+    displayName: string;
+    threads: {
+        [key: string]: ChatThreadClientState;
+    };
+    latestErrors: ChatErrors;
+};
+
+// @public (undocumented)
+export const ChatComposite: (props: ChatCompositeProps) => JSX.Element;
+
+// @public (undocumented)
+export type ChatCompositeClientState = {
+    userId: string;
+    displayName: string;
+    thread: ChatThreadClientState;
+    latestErrors: ChatAdapterErrors;
+};
+
+// @public (undocumented)
+export type ChatCompositeProps = {
+    adapter: ChatAdapter;
+    fluentTheme?: PartialTheme | Theme;
+    rtl?: boolean;
+    locale?: Locale;
+    onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+    onRenderMessage?: (messageProps: MessageProps, defaultOnRender?: DefaultMessageRendererType) => JSX.Element;
+    onRenderTypingIndicator?: (typingUsers: CommunicationParticipant[]) => JSX.Element;
+    options?: ChatOptions;
+    identifiers?: Identifiers;
+};
+
+// @public
+export class ChatError extends Error {
+    constructor(target: ChatErrorTarget, inner: Error);
+    inner: Error;
+    target: ChatErrorTarget;
+}
+
+// @public
+export const chatErrorBarSelector: OutputSelector<ChatClientState, {
+activeErrors: ErrorType[];
+}, (res: ChatErrors) => {
+activeErrors: ErrorType[];
+}>;
+
+// @public
+export type ChatErrorListener = (event: {
+    operation: string;
+    error: Error;
+}) => void;
+
+// @public
+export type ChatErrors = {
+    [target in ChatErrorTarget]: Error;
+};
+
+// @public
+export type ChatErrorTarget = 'ChatClient.createChatThread' | 'ChatClient.deleteChatThread' | 'ChatClient.getChatThreadClient' | 'ChatClient.listChatThreads' | 'ChatClient.off' | 'ChatClient.on' | 'ChatClient.startRealtimeNotifications' | 'ChatClient.stopRealtimeNotifications' | 'ChatThreadClient.addParticipants' | 'ChatThreadClient.deleteMessage' | 'ChatThreadClient.getMessage' | 'ChatThreadClient.getProperties' | 'ChatThreadClient.listMessages' | 'ChatThreadClient.listParticipants' | 'ChatThreadClient.listReadReceipts' | 'ChatThreadClient.removeParticipant' | 'ChatThreadClient.sendMessage' | 'ChatThreadClient.sendReadReceipt' | 'ChatThreadClient.sendTypingNotification' | 'ChatThreadClient.updateMessage' | 'ChatThreadClient.updateTopic';
+
+// @public (undocumented)
+export type ChatMessage = Message<'chat'>;
```

Comment:
This conflicts with the imported type `ChatMessage` - can we avoid it? (why we see ChatMessage_2 throughout this doc)